test/cli/.rubocop.yml

```diff
@@ -0,0 +1,27 @@
+# KBSecret style enforcement.
+# overrides for CLI tests
+
+inherit_from: ../../.rubocop.yml
+
+# we are organizing our tests in one class per command
+# this necessitates some long classes
+Metrics/ClassLength:
+  Enabled: false
+
+# we are organizing many of our tests in one method per option
+# this necessitates some long methods
+Metrics/MethodLength:
+  Enabled: false
+
+# see above
+# this necessitates some complex methods
+Metrics/AbcSize:
+  Enabled: false
+
+# see above
+# this necessitates some complex methods
+Metrics/CyclomaticComplexity:
+  Enabled: false
+
+Style/RegexpLiteral:
```

Comment:
Don't disable this check -- if you need inner slashes in a regexp, the `%r{}` literal syntax will be useful.

Reply:
My thinking here is to silence complaints about existing code like this: 

```
  def test_session
    ...
    error_pattern_no_record = /No such record\(s\)/
    error_pattern_no_session = /Unknown session/
    ...
```
...where we are favoring the consistency of a single syntax for regexes at the cost of backslashes. I think what we have done reads better than mixing `/pattern/` and `%r{pattern}` throughout the code and definitely don't want to go back and change all the patterns that happen to contain a backslash. If we are happy with the code the way it is, then it makes sense for our rubocop rules to reflect that, doesn't it?

Reply:
Yeah, that's reasonable. You can skip this, then.